tests/subworkflows/nf-core/fastq_contam_seqtk_kraken/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: SEQTK_SAMPLE {
+        publishDir = [ enabled: false ]
+    }
+    withName: UNTAR {
+        publishDir = [ enabled: false ]
+    }
```

Comment:
why?

Reply:
Otherwise the kx2d stuff ends up in the `test.yml`too - probably not desired to keep the test.yml as short as possible since we're testing the reporting and not that Untar works? 

Can also add it back

Reply:
yes, that's from the test.config part

Reply:
I also tend to disable "supporting" processes to keep `test.yml` relevant